src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -511,10 +510,9 @@ public boolean hasCollisionsIn(@NotNull BoundingBox boundingBox)
 
 	@Override
 	@Deprecated
-	public @NotNull Block getBlockAtKey(long key)
+	public @NotNull BlockMock getBlockAtKey(long key)
```

Comment:
Is there a reason the return type here was changed?

Reply:
Just wanted to align with the other block methods we currently provide. Methods like `assertType` on BlockMock could come handy in tests. I really dont mind and will let you decide on witch fits the best

Reply:
sure lets keep the old signature 👍